src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/resources/MacResources.properties

```diff
@@ -42,6 +42,7 @@ error.tool.failed.with.output=Error: "{0}" failed with following output:
 resource.bundle-config-file=Bundle config file
 resource.app-info-plist=Application Info.plist
 resource.runtime-info-plist=Java Runtime Info.plist
+resource.runtime-image-info-plist=Java Runtime Bundle Info.plist
```

Comment:
`resource.runtime-bundle-info-plist` would be a better name as the list file belongs to a bundle, not the image.

Reply:
Fixed.